jquery.mCustomScrollbar.js

```diff
@@ -1455,7 +1455,7 @@ and dependencies (minified).
 						dlt=e.deltaY || delta;
 				}
 				if((dir==="y" && !d.overflowed[0]) || (dir==="x" && !d.overflowed[1])){return;}
-				if(o.mouseWheel.invert){dlt=-dlt;}
+				if(o.mouseWheel.invert || (o.horizontalScroll && e.deltaX !== -0)){dlt=-dlt;}
```

Comment:
o.horizontalScroll  : that mean scroll has horizontal.

e.deltaX !== -0 : that mean mousewheel has not Y roll.